app/assets/javascripts/test/api/api_skeleton_latest.spec.js

```diff
@@ -228,6 +228,16 @@ test.serial("getTreeGroups should get all tree groups and set a tree group", t =
   t.is(state.tracing.skeleton.trees[1].groupId, 7);
 });
 
+test.serial("getTreeGroups should get all tree groups and set a tree group", t => {
```

Comment:
The test title was not adapted, should be sth like "renameGroup should rename a tree group"